public/index.html

```diff
@@ -31,6 +31,12 @@
       integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD"
       crossorigin="anonymous"
     />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
```

Comment:
why do we have 2 versions of bootstrap? `bootstrap@5.0.2` and `bootstrap@5.3.0-alpha1`?

Reply:
I have removed unwanted version